README.md

```diff
@@ -206,3 +221,26 @@ Next proposed features
 ----------------------
 * Implement `stringTweak` and `stringOptionalTweak` to better discriminate where the tweak is is optional or not.
   * experiment generic tweak definition: `tweak<String>` and `tweak<String?>`
+
+
+How to migrate from 0.20.4 to 0.21.0?
+-------------------------------------
+
+You need to remove the line bellow from your `Application` class:
```

Comment:
This information has to be updated with the latest changes

Reply:
Sorry, I forgot this section. Thanks so much